tests/phpunit/tests/functions/wpuploadbits.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Test the wp_upload_bits function
+ *
+ * @group Functions
+ * @group Upload
+ * @covers ::wp_upload_bits
```

Comment:
```suggestion
 * Tests the wp_upload_bits() function.
 *
 * @group functions.php
 * @group upload
 *
 * @covers ::wp_upload_bits
```
- Docblock descriptions should use third-person singular verbs (i.e. "Tests", not "Test").
- The correct group name is `functions.php`.
- The correct group name is `upload`.
- See [this ticket](https://core.trac.wordpress.org/ticket/56070) (the second code block) for the ordering and spacing for annotations in the test suite.